libcst/metadata/tests/test_expression_context_provider.py

```diff
@@ -411,3 +412,21 @@ def test_function(self) -> None:
                 },
             )
         )
+
+    def test_walrus(self) -> None:
+        if sys.version_info < (3, 8):
+            self.skipTest("This python version does not support :=")
+        code = """
+        if x := y:
+            pass
+        """
+        wrapper = MetadataWrapper(parse_module(dedent(code)))
```

Comment:
The version can be decoupled; if you want, this will let the test run everywhere:

```
parse_module(
  dedent(code),
  config=cst.PartialParserConfig(python_version="3.8"),
)
```